react-native-libraries.json

```diff
@@ -9312,5 +9312,16 @@
     "examples": ["https://github.com/Malaa-tech/react-native-simple-line-chart/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/sacmii/rn-vertical-slider",
+    "npmPkg": "rn-vertical-slider",
+    "nameOverride": "rn-vertical-slider",
```

Comment:
Hello @sacmii, thank you for adding new library to the Directory! 👍 

```suggestion
```

Those fields can be safely removed, besides that the entry LGTM.